modules/ossm-control-plane-remove.adoc

```diff
@@ -0,0 +1,70 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/service_mesh_install/removing-ossm.adoc
+
+[id="ossm-control-plane-remove_{context}"]
+= Removing the {ProductName} control plane
+
+You can remove the {ProductShortName} control plane by using the {product-title} web console or the CLI.
+
+
+[id="ossm-control-plane-remove-operatorhub_{context}"]
+== Removing the control plane with the web console
+
+Follow this procedure to remove the {ProductName} control plane by using the web console.
+
+.Prerequisites
+
+* The {ProductName} control plane must be deployed.
+
+.Procedure
+
+. Log into the {product-title} web console.
+
+. Navigate to *Catalogs* -> *Installed Operators*.
```

Comment:
This does not seem correct.  Line 26 should come before this step.

Reply:
Fixed.